wikimedia/wikipedia-ios
# [YiR] Update flow to show collective for logged out users

**Phabricator:** https://phabricator.wikimedia.org/T379796

### Notes
This PR removes the requirement that a user must be eligible to see at least one personalized slide for the entry point to appear. It also defaults to all collective slides from the coordinator if the user is logged out.

### Test Steps
1. Ensure device/simulator is in target region. Log out of app if needed, then fresh install app to ensure a logout state upon first launch.
2. Year in Review feature announcement should display when landing on Explore.
3. Go through Year in Review flow, you should see all collective slides.
4. Once you are done with the flow, go to Profile, tap Year in Review button.
5. You should see login/account creation prompt (from https://phabricator.wikimedia.org/T379794 work).
6. Log in, you should now be able to enter the flow through Profile.
7. Depending on your account, you will see the personalized editing slide if you have edits, or the collective slide. Other slides should still be collective since you didn't build up an article history.
8. Once complete with YiR flow, go to app Settings and turn off Year in Review.
9. Go through articles and build up a viewed history. Save some articles as well.
10. Go back to app Settings and turn on Year in Review.
11. Go through Year in Review flow through Profile, you should now see reading personalized slides.
